content/quickstart/java/tracing.md

```diff
@@ -1040,7 +1044,7 @@ public class Repl {
         }
     }
 
-    private static void readEvaluateProcess(BufferedReader in) throws IOException {
+    private static void repl(BufferedReader in) throws IOException {
```

Comment:
Java likes long names for methods in general. Better to use readEvaluateProcessLine?

Reply:
Aye aye, I actually thought of doing that but shortened it for Go and Python :) Let me update that in a second, thanks!